graphos/templates/graphos/gchart/base.html

```diff
@@ -2,7 +2,12 @@
 <script type="text/javascript">
   google.setOnLoadCallback(drawChart{{ chart.get_html_id }});
   function drawChart{{ chart.get_html_id }}() {
-    var data = google.visualization.arrayToDataTable({{ chart.get_data_json|safe }});
+    {% if chart.get_chart_type == 'candle_stick' %}
+        var data = google.visualization.arrayToDataTable({{ chart.get_data_json|safe }}, true);
+    {% else %}
+        var data = google.visualization.arrayToDataTable({{ chart.get_data_json|safe }});
+
+    {% endif %}
```

Comment:
@rameshugar This change shouldn't be made in base.html. Any candlestick specific thing should be done in gchart/candlestick_chart.html. And I believe this thing is already taken care of in gchart/candlestick_chart.html.